sample/src/test/java/com/microsoft/gctoolkit/sample/test/TestMain.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.gctoolkit.sample.test;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.microsoft.gctoolkit.sample.Main;
+
+public class TestMain {
+
+    @Test
```

Comment:
This is not a test, this is a runner.

Reply:
Use JUnit's assertNoException syntax.  I forget exactly what it is.

Reply:
Fixed in 22de7a7